src/Donations/Micros/DonationTypes/BouquetDonations.tsx

```diff
@@ -98,11 +98,7 @@ function FundingDonations({ setopenCurrencyModal }: Props): ReactElement {
                 </div> */}
                 <div className="funding-selection-option-text">
                   <span style={{ fontSize: "20px" }}>
-                    {getFormatedCurrency(
-                      i18n.language,
-                      currency,
-                      paymentSetup.unitCost * option.quantity
-                    )}
+                    {paymentSetup.unitCost * option.quantity}
```

Comment:
I would propose to just do:
```
{getFormatedCurrency(
  i18n.language,
  '',
  paymentSetup.unitCost * option.quantity
)}
```
<img width="823" alt="Bildschirmfoto 2021-11-17 um 13 39 23" src="https://user-images.githubusercontent.com/1532418/142202253-417d9f0f-8615-47d4-9c7b-765e202621ba.png">